acceptance: TestDockerCLI_test_missing_log_output failed

acceptance.TestDockerCLI_test_missing_log_output [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10963939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10963939?buildTab=artifacts#/) on master @ [66f234a67d347c95536715771266d34de08466ad](https://github.com/cockroachdb/cockroach/commits/66f234a67d347c95536715771266d34de08466ad):


```
=== RUN   TestDockerCLI_test_missing_log_output
    test_log_scope.go:167: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output3076630267
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestDockerCLI_test_missing_log_output
    generated_cli_test.go:306: -- test log scope end --
--- FAIL: TestDockerCLI_test_missing_log_output (55.46s)
=== RUN   TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
=== CONT  TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
    util_docker.go:254: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl (49.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_missing_log_output.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI_test_missing_log_output [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10964706?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10964706?buildTab=artifacts#/) on master @ [cd0381fc097245106382b4704ab42bf0470c2364](https://github.com/cockroachdb/cockroach/commits/cd0381fc097245106382b4704ab42bf0470c2364):


```
=== RUN   TestDockerCLI_test_missing_log_output
    test_log_scope.go:167: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output159348824
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestDockerCLI_test_missing_log_output
    generated_cli_test.go:306: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output159348824
--- FAIL: TestDockerCLI_test_missing_log_output (56.16s)
=== RUN   TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
=== CONT  TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
    util_docker.go:254: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl (50.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_missing_log_output.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI_test_missing_log_output [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10965223?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10965223?buildTab=artifacts#/) on master @ [06b5ba7888339ed53477ff73f9f3ae77a708ee17](https://github.com/cockroachdb/cockroach/commits/06b5ba7888339ed53477ff73f9f3ae77a708ee17):


```
=== RUN   TestDockerCLI_test_missing_log_output
    test_log_scope.go:167: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output919726859
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestDockerCLI_test_missing_log_output
    generated_cli_test.go:306: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output919726859
--- FAIL: TestDockerCLI_test_missing_log_output (57.01s)
=== RUN   TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
=== CONT  TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
    util_docker.go:254: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl (51.33s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_missing_log_output.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI_test_missing_log_output [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10972345?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/10972345?buildTab=artifacts#/) on master @ [b2ba2e537141fb4f836c3ba18ed6238e41f916c4](https://github.com/cockroachdb/cockroach/commits/b2ba2e537141fb4f836c3ba18ed6238e41f916c4):


```
=== RUN   TestDockerCLI_test_missing_log_output
    test_log_scope.go:167: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output2506450677
    test_log_scope.go:81: use -show-logs to present logs inline
=== CONT  TestDockerCLI_test_missing_log_output
    generated_cli_test.go:306: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/55708e7642e05278c2e0c2628af481fa/logTestDockerCLI_test_missing_log_output2506450677
--- FAIL: TestDockerCLI_test_missing_log_output (54.11s)
=== RUN   TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
=== CONT  TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl
    util_docker.go:254: non-zero exit code: 1
    --- FAIL: TestDockerCLI_test_missing_log_output/test_missing_log_output.tcl (48.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI_test_missing_log_output.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/acceptance/generated_cli_test.go
+++ b/pkg/acceptance/generated_cli_test.go
@@ -298,6 +298,13 @@ func TestDockerCLI_test_log_flags(t *testing.T) {
 	runTestDockerCLI(t, "test_log_flags", "../cli/interactive_tests/test_log_flags.tcl")
 }
 
+func TestDockerCLI_test_missing_log_output(t *testing.T) {
+	s := log.Scope(t)
+	defer s.Close(t)
+
+	runTestDockerCLI(t, "test_missing_log_output", "../cli/interactive_tests/test_missing_log_output.tcl")
+}
+
 func TestDockerCLI_test_multiline_statements(t *testing.T) {
 	s := log.Scope(t)
 	defer s.Close(t)

--- a/pkg/cli/interactive_tests/test_missing_log_output.tcl
+++ b/pkg/cli/interactive_tests/test_missing_log_output.tcl
@@ -1,7 +1,5 @@
 #! /usr/bin/env expect -f
 
-# disabled until #107122 is resolved.
-
 source [file join [file dirname $argv0] common.tcl]
 
 spawn /bin/bash
@@ -71,6 +69,7 @@ end_test
 
 # Stop it.
 interrupt
+interrupt
 eexpect ":/# "
 
 start_test "Check that --logtostderr can override the threshold but no error is printed on startup"
@@ -79,6 +78,7 @@ eexpect "marker\r\nCockroachDB node starting"
 end_test
 
 # Stop it.
+interrupt
 interrupt
 eexpect ":/# "
 

